bambinomio/assets/html/pulveris.html

```diff
@@ -1,54 +1,58 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-<style>
-.card {
-  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
-  max-width: 700px;
-  margin: auto;
-  text-align: center;
-  font-family: arial;
-}
-
-.price {
-  color: grey;
-  font-size: 22px;
-}
-
-.card button {
-  border: none;
-  outline: 0;
-  padding: 12px;
-  color: white;
-  background-color: #000;
-  text-align: center;
-  cursor: pointer;
-  width: 100%;
-  font-size: 30px;
-}
-
-.card button:hover {
-  opacity: 0.7;
-}
-</style>
+  <div id="pulveris" class="w3-padding-large"></div>
+  <style>
+    .card {
+      box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+      max-width: 1000px;
+      margin: auto;
+      text-align: center;
+      font-family: arial;
+    }
+
+    .price {
+      color: grey;
+      font-size: 22px;
+    }
+
+    .card button {

```

Comment:
card ir klases vārds. Nodefinējot šo klasi otrreiz, tiek anulēta pirmā definīcija. Kas īsti ir mēģināts panākt ar pirmo un kas ar otro?